SEC/Debt/AssetBackedSecurities/CDOs.rdf

```diff
@@ -384,34 +373,24 @@
 		<owl:disjointWith rdf:resource="&fibo-sec-dbt-ab-cmo;AgencyCMO"/>
 		<skos:definition xml:lang="en">structured debt security that has investment-grade bonds as its underlying assets backed by the receivables on high-yield or junk bonds</skos:definition>
+		<fibo-fnd-utl-av:abbreviation xml:lang="en">CBO</fibo-fnd-utl-av:abbreviation>
 		<fibo-fnd-utl-av:explanatoryNote xml:lang="en">A multitranche debt structure similar in some respects to a collateralized mortgage obligation (CMO) structure. Typically low-rated bonds rather than mortgages serve as the collateral. The organization creating and promoting the structure usually holds the underlying equity and may also collect a fee. Junk bonds are typically not investment grade, but because they pool several types of credit quality bonds together, they offer enough diversification to be &quot;investment grade.&quot; For example high yield [emerging market] CBO which consists of a portfolio of different high yield [emerging market] bonds. Investopedia: Similar in structure to a collateralized mortgage obligation (CMO), but different in that CBOs represent different levels of credit risk, not different maturities. Defoinition Origin:Investopedia</fibo-fnd-utl-av:explanatoryNote>
 	</owl:Class>
 	
-	<owl:Class rdf:about="&fibo-sec-dbt-ab-cdo;CollateralizedBondObligationDeal">
-		<rdfs:subClassOf rdf:resource="&fibo-sec-dbt-ab-pbs;CollateralizedObligationDeal"/>
-		<rdfs:label xml:lang="en">collateralized bond obligation deal</rdfs:label>
-		<skos:definition xml:lang="en">An issue of Collateralized Bond Obligation notes.</skos:definition>
-	</owl:Class>
-	
-	<owl:Class rdf:about="&fibo-sec-dbt-ab-cdo;CollateralizedLoanObligation">
-		<rdfs:subClassOf rdf:resource="&fibo-sec-dbt-ab-pbs;CollateralizedObligationInstrument"/>
+	<owl:Class rdf:about="&fibo-sec-dbt-ab-cdo;CollateralizedLoanObligationOffering">
+		<rdfs:subClassOf rdf:resource="&fibo-sec-dbt-ab-pbs;CollateralizedObligationOffering"/>
 		<rdfs:subClassOf>
 			<owl:Restriction>
 				<owl:onProperty rdf:resource="&fibo-fnd-rel-rel;refersTo"/>
 				<owl:someValuesFrom rdf:resource="&fibo-sec-dbt-ab-abs;LoanPool"/>
 			</owl:Restriction>
 		</rdfs:subClassOf>
-		<rdfs:label xml:lang="en">collateralized loan obligation</rdfs:label>
+		<rdfs:label xml:lang="en">collateralized loan obligation offering</rdfs:label>
 		<owl:disjointWith rdf:resource="&fibo-sec-dbt-ab-cmo;AgencyCMO"/>
-	</owl:Class>
-	
-	<owl:Class rdf:about="&fibo-sec-dbt-ab-cdo;CollateralizedLoanObligationDeal">
-		<rdfs:subClassOf rdf:resource="&fibo-sec-dbt-ab-pbs;CollateralizedObligationDeal"/>
-		<rdfs:label xml:lang="en">collateralized loan obligation deal</rdfs:label>
-		<skos:definition xml:lang="en">An issue of Collateralized Loan Obligations.</skos:definition>
+		<fibo-fnd-utl-av:abbreviation xml:lang="en">CLO offering</fibo-fnd-utl-av:abbreviation>
 	</owl:Class>
 	
 	<owl:Class rdf:about="&fibo-sec-dbt-ab-cdo;Creditworthiness">
```

Comment:
need to have individuals not a set of string values. And reconcile with the Ratings ontology